tests/crm/test_crm.py

```diff
@@ -888,12 +889,18 @@ def test_crm_fdb_entry(duthosts, rand_one_dut_hostname, tbinfo):
 
     # Make sure CRM counters updated
     time.sleep(CRM_UPDATE_TIME)
-
-    # Get new "crm_stats_fdb_entry" used and available counter value
-    new_crm_stats_fdb_entry_used, new_crm_stats_fdb_entry_available = get_crm_stats(get_fdb_stats, duthost)
+    # Timeout for asyc fdb clear 
+    FDB_CLEAR_TIMEOUT = 10
+    while FDB_CLEAR_TIMEOUT > 0:
+        # Get new "crm_stats_fdb_entry" used and available counter value
+        new_crm_stats_fdb_entry_used, new_crm_stats_fdb_entry_available = get_crm_stats(get_fdb_stats, duthost)
+        if new_crm_stats_fdb_entry_used == 0:
+            break
+        FDB_CLEAR_TIMEOUT -= CRM_POLLING_INTERVAL
+        time.sleep(CRM_POLLING_INTERVAL)
```

Comment:
Can we do a `wait_until` here instead?

Reply:
Good suggestion. I didn't use ```wait_until``` because I think the current code is easier to read and understand. Or a new wraper function is needed to check the return value of get_crm_stats